js/wp-seo-admin.js

```diff
@@ -31,11 +37,11 @@ jQuery(document).ready(function () {
         jQuery(this).addClass('nav-tab-active');
     });
 
-    jQuery("#company_or_person").change(function () {
-        if ('company' == jQuery(this).val()) {
+    jQuery('#company_or_person').change(function () {
+        if ('company' === jQuery(this).val()) {
             jQuery('#knowledge-graph-company').show();
             jQuery('#knowledge-graph-person').hide();
-        } else if ('person' == jQuery(this).val()) {
+        } else if ('person' === jQuery(this).val()) {
             jQuery('#knowledge-graph-company').hide();
```

Comment:
Assigning jQuery(this).val() to a var and then use this in the if-statements on line 41 and 44 is a bit quicker than running the val()-function twice.